pkgs/tools/networking/wireguard-go/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildGoPackage, fetchurl }:
+
+buildGoPackage rec {
+  name = "wireguard-go-${version}";
+  version = "0.0.20180514";
+
+  goPackagePath = "wireguard-go";
+
+  src = fetchurl {
+    url = "https://git.zx2c4.com/wireguard-go/snapshot/wireguard-go-${version}.tar.xz";
+    sha256 = "1bn49a67m2ab0l9lq3zh2mfbbppmyg34klqi3069sjn6lg2hlajs";
+  };
+
+  goDeps = ./deps.nix;
+
+  patchPhase = ''
+    # Replace local imports so that go tools do not trip on them
+    find . -name '*.go' -exec sed -i '/import (/,/)/s@"./@"${goPackagePath}/@' {} \;
```

Comment:
I think you've used the wrong tool to generate `deps.nix`. Try dep2nix: https://github.com/nixcloud/dep2nix

Reply:
I am pretty sure this has nothing to do with the issue. The problem is that `go build` and `go install` do not work with relative imports (e.g. https://stackoverflow.com/a/34854318/603094).

I tried `dep2nix` nevertheless and the only difference is the header line.

Reply:
Yes, sorry, commented on the wrong line.